ports/espressif/boards/m5stack_m5paper/mpconfigboard.mk

```diff
@@ -0,0 +1,9 @@
+CIRCUITPY_CREATOR_ID =  0x10151015
+CIRCUITPY_CREATION_ID = 0x0032000B
```

Comment:
Please PR this id here: https://github.com/creationid/creators/blob/main/creations/m5stack.md

Reply:
Added PR creationid/creators#44